fieldbehavior/required.go

```diff
@@ -76,7 +76,16 @@ func validateRequiredFields(reflectMessage protoreflect.Message, mask *fieldmask
 	return nil
 }
 
+func isEmpty(mask *fieldmaskpb.FieldMask) bool {
+	paths := mask.GetPaths()
+	return mask == nil ||
+		len(paths) == 0
```

Comment:
Nitpick, so feel free to ignore: If we're not afraid of `mask.GetPath()` triggering a nil-pointer dereferencing (I'd say we're not), then this should be enough IMO.
```suggestion
	return len(mask.GetPaths()) == 0
```

Reply:
agree, we are not afraid of this nil-pointer. Changed it :smile: